vm.c

```diff
@@ -63,13 +63,13 @@ static inline void vm_push(vm_env *env, size_t n);
 #define VM_JGT() VM_J_TYPE_INST(>)
 #define VM_JNZ() VM_J_TYPE_INST(!=)
 
-#define VM_CALL_HANDLER()                                                      \
-    do {                                                                       \
-        if (OPCODE_IMPL(OPCODE))                                               \
-            OPCODE_IMPL(OPCODE)                                                \
-        (vm_get_op_value(env, &OPCODE.op1), vm_get_op_value(env, &OPCODE.op2), \
-         vm_get_temp_value(env, OPCODE.result));                               \
-        DISPATCH;                                                              \
+#define VM_CALL_HANDLER()                                                \
```

Comment:
Use `/* clang-format off */` and `/* clang-format on */` to perform manual indentation explicitly.